spring-boot/proxy/src/main/java/eu/coatrack/proxy/security/ApiKeyAuthTokenVerifier.java

```diff
@@ -21,58 +21,36 @@
  */
 
 import eu.coatrack.api.ApiKey;
-import eu.coatrack.api.Proxy;
 import eu.coatrack.api.ServiceApi;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.annotation.Value;
-import org.springframework.http.HttpStatus;
-import org.springframework.http.ResponseEntity;
-import org.springframework.security.access.AccessDeniedException;
 import org.springframework.security.authentication.AuthenticationManager;
-import org.springframework.security.authentication.AuthenticationServiceException;
 import org.springframework.security.authentication.BadCredentialsException;
-import org.springframework.security.authentication.CredentialsExpiredException;
 import org.springframework.security.core.Authentication;
 import org.springframework.security.core.AuthenticationException;
 import org.springframework.security.core.authority.SimpleGrantedAuthority;
+import org.springframework.stereotype.Service;
 import org.springframework.util.Assert;
-import org.springframework.web.client.HttpClientErrorException;
-import org.springframework.web.client.RestTemplate;
 
-import java.sql.Date;
-import java.time.LocalDate;
-import java.util.Collections;
-import java.util.HashSet;
-import java.util.Set;
-
-import org.springframework.http.client.support.BasicAuthorizationInterceptor;
+import java.util.*;
 
 /**
- * Checks if the API key token value sent by the client is valid. Right now this
- * is done by a request to CoatRack admin, asking if the token value is known.
+ *  Checks if the API key token value sent by the client is valid. If so, the client
+ *  is forwarded to the requested service API.
```

Comment:
This class is only a verifier, it "does not need to know" about the consequences of its check. 

Therefore, maybe the comment in this class should not explain business logic by other classes (as maybe those other classes will be changed in the future).

Reply:
Done.